SolarCarTelemetry.sublime-workspace

```diff
@@ -0,0 +1,216 @@
+{
```

Comment:
I think this file was accidentally added. If you want you can add a gitignore for *.sublime-workspace.